File: src/main/java/org/lwjgl/Sys.java
package org.lwjgl;

@SuppressWarnings("unused")
public class Sys {
    public static final String VERSION = Version.getVersion();

    public static long getTime() {
        return System.nanoTime();
    }

    public static long getTimerResolution() {
        return 1_000_000_000;
    }

    public static void initialize() {
    }

    public static String getVersion() {
        return VERSION;
    }
}

File: lwjgl/src/main/kotlin/org/lwjgl/Sys.kt
package org.lwjgl

import org.lwjgl.glfw.GLFWErrorCallback

@Suppress("unused")
object Sys {
    private var initialized = false

    @JvmStatic
    fun getVersion(): String = Version.getVersion()

    @JvmStatic
    fun getTime(): Long = System.nanoTime()

    @JvmStatic
    fun getTimerResolution(): Long = 1000000000

    @JvmStatic
    fun initialize() {
        if (initialized) return

        GLFWErrorCallback.createThrow().set()
    }
}
